docs/documentation/install/introduction.md

```diff
@@ -1,6 +1,6 @@
 # Installation guide for new users
 
-This guide will walk you through installing and getting started with the kit. You don’t need any technical knowledge to follow along. If you get stuck, post a message on Slack, or if you have a developer on your team, they should be able to help.
+This guide will walk you through installing and getting started with the kit. You don’t need any technical knowledge to follow along. If you get stuck, contact us using the [#prototype-kit Slack channel](https://ukgovernmentdigital.slack.com/messages/C0647LW4R/convo/C41GC5UAY-1506957241.000096/), or if you have a developer on your team, they should be able to help.
```

Comment:
That link's not quite right – it's linking to the #prototype-kit channel, but with a conversation from #apis open as well!

I think it should just be:

```suggestion
This guide will walk you through installing and getting started with the kit. You don’t need any technical knowledge to follow along. If you get stuck, contact us using the [#prototype-kit Slack channel](https://ukgovernmentdigital.slack.com/messages/C0647LW4R/), or if you have a developer on your team, they should be able to help.
```

Reply:
oh gosh sorry I missed that yeh